alcs-frontend/src/app/services/home/home.service.ts

```diff
@@ -22,7 +22,7 @@ export class HomeService {
         reconsiderations: ApplicationReconsiderationDto[];
         planningReviews: PlanningReviewDto[];
         modifications: ApplicationModificationDto[];
-        covenants: CovenantDto[];
+        covenants: NoticeOfIntentDto[];
```

Comment:
looks a bit confusing to have covenants of type NoticeOfIntentDto[]

Reply:
Fixed